### PR TITLE
Opt kin new

### PR DESCRIPTION
**Pull request by roystgnr**
Rebasing and conflict fixes for #95 

Diffs look good to me, we're compiling and passing make check on Antioch, GRINS, and FIN-S.

### CODE REVIEW

File: src/kinetics/include/antioch/reaction.h [hunk 1]
```diff
@@ -324,9 +324,9 @@ namespace Antioch
     std::vector<unsigned int> _reactant_stoichiometry;
     std::vector<unsigned int> _product_stoichiometry;
     std::vector<CoeffType> _efficiencies;
-    std::vector<unsigned int> _species_reactant_stoichiometry;
-    std::vector<unsigned int> _species_product_stoichiometry;
-    std::vector<int>          _species_delta_stoichiometry;
+//    std::vector<unsigned int> _species_reactant_stoichiometry;
```
Comment (pbauman) [inline src/kinetics/include/antioch/reaction.h]: Is there a reason we're leaving all this commented out stuff in?

File: src/kinetics/include/antioch/reaction.h [hunk 2]
```diff
@@ -638,42 +638,46 @@ namespace Antioch
...
+        /*_species_product_stoichiometry[this->product_id(p)] =
+          this->product_stoichiometric_coefficient(p);*/
+        _gamma += this->product_stoichiometric_coefficient(p);
       }
     
     // find the delta stoichiometric coefficient for each species,
     // and the sum of the deltas 
-    _gamma = 0;
-    for (unsigned int s=0; s<this->n_species(); s++)
+    // Sylvain: useless
```
Comment (pbauman) [inline src/kinetics/include/antioch/reaction.h]: If it's useless, why is it still here?

**Comment by pbauman** [comment]
Other than what appear to be stale code that's getting exorcised, I'm good.

**Comment by SylvainPlessis** [comment]
I didn't suppress definitively the code for ``verboser'' history.  I'm good for a cleaning commit then merging.

File: src/kinetics/include/antioch/reaction.h (hunk 1 above)
Comment (roystgnr) [inline src/kinetics/include/antioch/reaction.h]: Personally I didn't want to futz too much with Sylvain's commits.  I'd vote to delete too.

**Comment by roystgnr** [comment]
Since this is on my fork now I'll do the cleaning commit.  Thanks for the extra eyes, Paul.

**Pull request merged**